config/reporting_service.yml.dist

```diff
@@ -15,6 +15,9 @@ federationregistry:
 rapidconnect:
   host: rapid.test.aaf.edu.au
   secret: 'This is the shared secret used for authenticating to the Rapid export API'
+saml_service:
+  metadata_url: https://saml.test.url/mdq
+  # metadata_cert: "The Certificate"
```

Comment:
It wasn't clear to me right away whether this was meant to be a path to the certificate or the certificate embedded directly.  Probably obvious (and just not to me), but maybe the sample value could make it even clearer?

Reply:
As  configuration is now via env vars, which is not exactly suitable for passing full certificates, I'll change the setting to use path to certificate instead.